src/util.h

```diff
@@ -29,6 +29,9 @@
 #include <boost/signals2/signal.hpp>
 #include <boost/thread/exceptions.hpp>
 
+// Application startup time (used for uptime calculation)
+static const int64_t nStartupTime = GetTime();
```

Comment:
No problem John, I understood what you meant :) When I mentioned I would make the change I was referring to moving `nStartupTime` to `util.cpp`. I pushed a new commit with this change.